src/main/java/pb360/model/embedded/line.java

```diff
@@ -0,0 +1,24 @@
+package pb360.model.embedded;
+
+public class line {
```

Comment:
why we need to create a class for store only variable?

Reply:
My idea was wrong so i delete this too